log4j-layout-template-json/src/main/resources/GcpLayout.json

```diff
@@ -49,25 +55,15 @@
     "key": "span_id"
   },
   "logging.googleapis.com/trace_sampled": true,
-  "_exception": {
-    "class": {
-      "$resolver": "exception",
-      "field": "className"
-    },
-    "message": {
-      "$resolver": "exception",
-      "field": "message"
-    },
-    "stackTrace": {
-      "$resolver": "pattern",
-      "pattern": "%xEx"
-    }
+  "exception": {
+    "$resolver": "pattern",
+    "pattern": "%xEx"
   },
-  "_thread": {
+  "thread": {
     "$resolver": "thread",
     "field": "name"
   },
-  "_logger": {
+  "logger": {
```

Comment:
Those field start intentionally with an underscore `_`, so they don't collide with payload field that Google might introduce in the future.

Reply:
They are already used by Google, just not documented. For example in their Spring GCP libraries https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/cabb7a59fa6d49d280dd311a7f59ec4d922300e7/spring-cloud-gcp-logging/src/main/java/com/google/cloud/spring/logging/StackdriverJsonLayout.java#L286-L287. My idea was to match the format Google already produces officially by their Java libraries as much as possible.

Reply:
Sounds good to me!